### PR TITLE
fixed flow error on Travis

### PR DESCRIPTION
**Pull request by bestander**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ script:
   - if [[ "$TEST_TYPE" = objc ]]; then travis_retry ./scripts/objc-test.sh; fi
   - if [[ "$TEST_TYPE" = e2e-objc ]]; then travis_retry ./scripts/e2e-test.sh --ios; fi
   # temporarily disable flow check because it went mad on Travis
```
Comment (jeffmo) [inline .travis.yml]: Time to delete this comment?

Comment (bestander) [inline .travis.yml]: :+1:

**Comment by bestander** [comment]
@facebook-github-bot shipit

**Comment by bestander** [comment]
shipping it to get master green sooner

**Comment by bestander** [comment]
@facebook-github-bot shipit

**Comment by bestander** [comment]
@facebook-github-bot shipit

**Comment by bestander** [comment]
@facebook-github-bot shipit

**Comment by bestander** [comment]
@facebook-github-bot shipit

**Pull request closed without merging**